crd-generator/api/src/main/java/io/fabric8/crd/generator/visitor/AnnotatedPropertyPathDetector.java

```diff
@@ -45,46 +49,52 @@ public AnnotatedPropertyPathDetector(String prefix, String annotationName, List<
     this(prefix, annotationName, parents, new AtomicReference<>(Optional.empty()));
   }
 
-  public AnnotatedPropertyPathDetector(String prefix, String annotationName, List<Property> parents, AtomicReference<Optional<String>> reference) {
+  public AnnotatedPropertyPathDetector(String prefix, String annotationName, List<Property> parents,
+      AtomicReference<Optional<String>> reference) {
     this.prefix = prefix;
     this.annotationName = annotationName;
     this.parents = parents;
     this.reference = reference;
   }
 
+  private boolean excludePropertyProcessing(Property p) {
+    return p.getAnnotations().stream()
+        .anyMatch(ann -> ann.getClassRef().getFullyQualifiedName().equals(ANNOTATION_JSON_IGNORE));
+  }
+
```

Comment:
Maybe make this a utility method that could be shared?

Reply:
if we decide to move ahead with this PR in the current form I will 🙂